models/User.js

```diff
@@ -0,0 +1,31 @@
+const mongoose = require("mongoose");
+
+const UserSchema = new mongoose.Schema({
+    zprn: {
+        type: String,
```

Comment:
I think our `ZPRN` is all number and not string. I have no idea about staff `ZPRN`

Reply:
The schema given indicated it as String, so I used it as String

Reply:
Matching the exact length is only available with String, hence datatype is String

Reply:
We can keep the validation as a string and then apply a regex on it to accept only the digits. 
```
joi.string().length(14).regex(/^\d+$/)
``` 
this is what I found on searching.
So that means we can switch to number in the database.

Reply:
Okay, will make this change